scripts/deploy.py

```diff
@@ -160,9 +160,31 @@ def preprocess_release():
         new_assets_file.write(content)
 
 
+def check_dev_dependencies_are_up_to_date():
```

Comment:
`chek_` method should return a boolean value, I think this method downloads the required dependencies if required. (Maybe we should rename this method?)

Reply:
Yes, renamed.